docs/source/tutorials/SAR/SAR tutorial.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b3084a00",
+   "metadata": {},
+   "source": [
+    "<h1 style=\"font-size: 36px;\">SAR tutorial</h1>\n",
+    "\n",
+    "In this notebook, we introduce synergistic action representations (SAR) and demonstrate how to use these representations to facilitate the training of high-dimensional continuous control policies within MyoSuite.\n",
+    "\n",
+    "The primary purpose of this notebook is to equip the user to train with SAR for their own task specification(s) both within and beyond the musculoskeletal control paradigm.\n",
+    "\n",
+    "\n",
+    "# Table of contents\n",
+    "\n",
+    "- [Imports and utilities](#imports-and-utilities)\n",
+    "- [Walkthrough of SAR core functions](#SAR-core-functions)\n",
+    "- [Example 1: SAR x physiological locomotion](#TODO)\n",
+    "- [Example 2: SAR x physiological dexterity](#TODO)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f6b5eddd",
+   "metadata": {},
+   "source": [
+    "# Imports and utilities"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f6fe8e7",
+   "metadata": {},
+   "source": [
+    "First, we import the packages that SAR requires, including OpenAI gym, MyoSuite, stable-baselines3, and sci-kit learn. It is recommended to `pip install` any imports that are missing in your environment.\n",
```

Comment:
Why's OpenAI-Gym a requirement? But we don't install it. Should this come from the envs repo (MyoSuite), or is this a SAR requirement?

Reply:
fixed this language, no actual gym requirement is required/enforced